scripts/deploy-dev.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
This script is nasty. Once we migrate our API reference to the Astro site, we won't need it anymore and we'll be able to use yarn build directly